### ISSUE TITLE
Empty gene names in the xml version

### ISSUE THREAD

**Issue by OmarAshkar** [SysBioChalmers/Mouse-GEM#10]
### Description of the issue:
When I read the model I got an error and I cannot find any gene names. I am still trying the model so maybe there are other issues followed.

#### Expected feature/value/output:
Gene names

#### Current feature/value/output:
empty string

#### Reproducing these results:
```python
model = read_sbml_model('data/Mouse-GEM.xml')
## '' is not a valid SBML 'SId'.
## https://identifiers.org/taxonomy/ does not conform to 'http(s)://identifiers.org/collection/id' ## or'http(s)://identifiers.org/COLLECTION:id
```

**I hereby confirm that I have:**
- [x] Tested my code with [all requirements](https://github.com/SysBioChalmers/yeast-GEM#required-software---user) for running the model
- [x] Done this analysis in the `master` branch of the repository
- [x] Checked that a similar issue does not exist already
- [x] If needed, asked first in the [Gitter chat room](https://gitter.im/SysBioChalmers/yeast-GEM) about the issue

**Comment by Hao-Chalmers** [comment]
@OAshkar can you try `load_yaml_model` way of loading instead, see if this helps.

**Comment by OAshkar** [comment]
unfortunately. this does not work at all with the following message

```
ScannerError: mapping values are not allowed here
  in "data/Mouse-GEM.xml", line 15750, column 110
```

**Comment by Hao-Chalmers** [comment]
Sorry for the misunderstanding

**Comment by Hao-Chalmers** [comment]
@OAshkar what I suggested was to load the `yml` model file as such:

```python
model = load_yaml_model('data/Mouse-GEM.yml')
```

Sorry for the misunderstanding.

**Comment by OAshkar** [comment]
Alright! I imported the model without errors but still no gene names. For example 
```
model.genes.Adh1.name
```

**Comment by Hao-Chalmers** [comment]
The `Mouse-GEM` is developed by RAVEN and in compliance with its [model structure](https://github.com/SysBioChalmers/RAVEN/wiki/RAVEN-Model-Structure), which has only mandatory field `genes` for gene ids. We haven't include other alternative gene names (e.g. `geneShortNames`).

Can you please let us know why do you need gene names? if the information of gene ids is not enough?

**Comment by OAshkar** [comment]
It just makes it easier to interactively find gene names. But it's ok. Thank you for your time.

**Comment by Hao-Chalmers** [comment]
@OAshkar the problem here probably is providing multiple sets of gene ids (Ensembl, NCBI, symbol) in Mouse-GEM.

**Comment by mihai-sysbio** [comment]
Is there a timeline for the next release?

**Comment by mihai-sysbio** [comment]
@haowang-bioinfo pinging about this in case it may be planned for the next release

**Comment by haowang-bioinfo** [comment]
fixed in v1.5.0
